src/handlers/admin_v1_configs.cpp

```diff
@@ -23,15 +24,22 @@ struct RequestData {
 };
 
 RequestData ParseRequest(const userver::formats::json::Value &request) {
```

Comment:
Let's delete `RequestData` and use the code-generated structs directly

Reply:
OK, done